### PR TITLE
STY: De-privatize imported names

### PR DESCRIPTION
**Pull request by jbrockmendel**
Getting close to the end on these.

### CODE REVIEW

File: pandas/tests/computation/test_eval.py [hunk 1]
```diff
@@ -1921,7 +1921,7 @@ def test_invalid_parser():
 }
 
 
-@pytest.mark.parametrize("engine", _engines)
+@pytest.mark.parametrize("engine", ENGINES)
 @pytest.mark.parametrize("parser", _parsers)
```
Comment (jreback) [inline pandas/tests/computation/test_eval.py:1925]: should be PARSERS

**Comment by jreback** [comment]
failing linting somewhere

File: pandas/tests/computation/test_eval.py (hunk 1 above)
Comment (jbrockmendel) [reply pandas/tests/computation/test_eval.py:1925]: in this case _parsers is a defined a few lines up

**Comment by jbrockmendel** [comment]
its the _np_version_under_118 thing again, just pushed the fix here too

**Pull request merged**